airflow/providers/google/cloud/transfers/gdrive_to_gcs.py

```diff
@@ -85,38 +92,14 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
-        self.destination_bucket = destination_bucket
-        self.destination_object = destination_object
+        self.bucket_name = destination_bucket or bucket_name
+        self.object_name = destination_object or object_name
```

Comment:
```suggestion
        self.bucket_name = destination_bucket or bucket_name
        if destination_bucket:
            warnings.warn("`destination_bucket ` is deprecated please use `bucket_name`", DeprecationWarning)
            bucket_name = destination_bucket
        self.bucket_name = bucket_name
```
In this way users will get DeprecationWarning in runtime, WDYT?

Reply:
I agree. Good suggestion